### PR TITLE
qa/tasks: fix cephadm config image check

### PR DESCRIPTION
**Pull request by gekios**
Check container_image_name only if ceph cluster image is not pre-defined in config.
We shouldn't care about container_image_name if there cephadm or ceph already have image defined.

Signed-off-by: Georgios Kyratsas <gkyratsas@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by gekios** [comment]
tests passed: https://pulpito.ceph.com/gkyratsas-2020-07-03_16:56:02-rados:cephadm:-master-distro-basic-smithi/

File: qa/tasks/cephadm.py [hunk 1]
```diff
@@ -1144,17 +1144,18 @@ def task(ctx, config):
...
-    if not container_image_name:
-        raise Exception("Configuration error occurred. "
-                        "The 'image' value is undefined for 'cephadm' task. "
-                        "Please provide corresponding options in the task's "
-                        "config, task 'overrides', or teuthology 'defaults' "
-                        "section.")
 
     if not hasattr(ctx.ceph[cluster_name], 'image'):
         ctx.ceph[cluster_name].image = config.get('image')
+
```
Comment (kshtsk) [inline qa/tasks/cephadm.py:1150]: This line is not related to the fix.

Comment (gekios) [reply qa/tasks/cephadm.py]: Yeah, it was a typo...

**Comment by gekios** [comment]
jenkins test make check

**Comment by kshtsk** [comment]
@gekios maybe it would be better to changed commit description to something like:
```
Check container_image_name only if ceph cluster image is not pre-defined in config.
We shouldn't care about container_image_name if there cephadm or ceph already have image defined.
```

**Comment by callithea** [comment]
jenkins test dashboard backend

**Comment by callithea** [comment]
jenkins test dashboard backend
